python_ta/transforms/type_inference_visitor.py

```diff
@@ -258,17 +258,46 @@ def register_type_constraints_setter():
     return type_visitor
 
 
+def _populate_local_env(node):
+    """Helper to populate locals attributes in type environment of given node."""
+    for var_name in node.locals:
+        try:
+            var_value = node.type_environment.lookup_in_env(var_name)
+            node.type_environment.locals[var_name] = var_value
```

Comment:
This line is duplicated across the two blocks. This means it can be pulled out to be after the try-except.